docker/etc/nginx/custom/01_docstore_files.conf

```diff
@@ -0,0 +1,63 @@
+## Download handler.
+location ~ "^/files/([0-9a-f]{2})([0-9a-f]{2})([0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/([^/]+)(\.[0-9a-zA-Z]+)$" {
```

Comment:
I don't understand the `^/` match; we can't ever have files with a slash in their filename and the consumer should not be generating such URLs.

Reply:
@cafuego This is to make sure it only matches the pattern `/files/{uuid}/{name}.{ext}`. If we use something like `[^.]+` instead then `/files/1f4d4a9a-9013-40ed-a23b-445a3a76038e/thats/not/what/we/want.txt` for example would be a match.